org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/tests/__snapshots__/unitTests.js.snap

```diff
@@ -1,3 +1,3 @@
-exports[`GetPlatform 1`] = `"{\"opent2t\":{\"schema\":\"org.opent2t.sample.binaryswitch.superpopular\",\"translator\":\"opent2t-translator-com-insteon-binaryswitch\",\"controlId\":\"Mock Device\"},\"pi\":\"44b0a5fb-f8a2-0160-0fae-34978b82a172\",\"mnmn\":\"Insteon\",\"mnmo\":\"Binary Switch (Generic)\",\"n\":\"Mock Device\",\"rt\":[\"org.opent2t.sample.binaryswitch.superpopular\"],\"entities\":[{\"n\":\"Mock Device\",\"icv\":\"core.1.1.0\",\"dmv\":\"res.1.1.0\",\"rt\":[\"oic.d.smartplug\"],\"di\":\"f9604075-1a64-498b-ae9b-7436a63721ba\",\"resources\":[{\"href\":\"/power\",\"rt\":[\"oic.r.switch.binary\"],\"if\":[\"oic.if.a\",\"oic.if.baseline\"]},{\"href\":\"/connectionStatus\",\"rt\":[\"oic.r.mode\"],\"if\":[\"oic.if.s\",\"oic.if.baseline\"]}]}]}"`;
```

Comment:
These are for 'all plaform' unit test ( see PR #201 ). @danielrischar for comments.

---
In reply to: [105797293](https://github.com/openT2T/translators/pull/197#discussion_r105797293) [](ancestors = 105797293)